.github/workflows/transformer.yaml

```diff
@@ -62,5 +62,5 @@ jobs:
 
       - name: Upload to S3
         run: |
-          s3cmd sync --acl-public --delete-removed --mime-type application/json \
+          s3cmd sync --acl-public --mime-type application/json \
```

Comment:
I see why you think this could be a good idea. But we actually need to remove the files that are not uploaded, otherwise we can't filter things in the transformer, since they will still be on S3 :) WDYT?

Reply:
Ah I see, I had no idea. Yes that makes sense to me.